e2e_tests/tests/cluster/test_model_registry.py

```diff
@@ -29,6 +29,12 @@ def test_model_registry() -> None:
         assert mnist.metadata == db_model.metadata
         assert mnist.metadata == {"testing": "metadata"}
 
+        # Confirm we can look up a model by its ID
+        db_model = d.get_model_by_id(mnist.model_id)
+        assert db_model.name == "mnist"
+        db_model = d.get_model(str(mnist.model_id))
+        assert db_model.name == "mnist"
```

Comment:
What does this mean for the python API?

If the interface is really:
```python
def get_model(identifier: Union[str, int]):
  ...
```
Then why not make that explicit, and deprecate `get_model_by_id()`?

I dunno, that seems like a sloppy interface.  But I guess if both types of key are guaranteed unique, then it's probably ok.

Reply:
I agree that it's not great to have one API method for both, but I can modify this to accept both types. I don't want to remove `get_model_by_id` in case someone updated their code to use this, but I will have it just call `get_model`